### PR TITLE
Update the SDK Manager version

### PR DESCRIPTION
**Pull request by graugans**
The latest 0.9.14-4964 is used for installation. The definition
of the version was moved to a variable and one common variable for the .deb
file is used.

Signed-off-by: Christian Ege <christian.ege@ifm.com>

### CODE REVIEW

**Comment by deltaflyer** [comment]
Hello @graugans 

thank your for handing in this PR. I am also looking forward for the new SDK versions to get my Board Support Package updated on the Jetson Nano.
@atinfinity do you mind merging this PR?

Best,
Oliver

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This is a Dockerfile to use [NVIDIA SDK Manager](https://docs.nvidia.com/sdk-man
 ### Download NVIDIA SDK Manager
 Please download the package of NVIDIA SDK Manager from <https://developer.nvidia.com/embedded/dlc/nv-sdk-manager>.  
 And, please put the package of NVIDIA SDK Manager in the same directory as the Dockerfile.  
-This time, I used `sdkmanager_0.9.12-4180_amd64.deb`.
+This time, I used `sdkmanager_0.9.14-4964_amd64.deb`.
 
 ### Build Docker image
```
Comment (atinfinity) [inline README.md:15]: @graugans Thank you for your proposal! I think that your change is good.
I hope you add the procedure to specify `SDK_MANAGER_VERSION` as of `docker build` to README.md.

**Review by atinfinity — CHANGES_REQUESTED** [review]

**Comment by atinfinity** [comment]
@deltaflyer Thank you for your comment.

> do you mind merging this PR?

I agree this proposal.

File: README.md (hunk 1 above)
Comment (graugans) [reply README.md:15]: @atinfinity sorry for the late response. I have not yet added advise how to set this from the docker command line. I will add this after cleaning my TODO list.

Comment (atinfinity) [reply README.md:15]: @graugans 

> I have not yet added advise how to set this from the docker command line.

I tried to set `SDK_MANAGER_VERSION` using `--build-arg`.  
It works well.

```shell
$ docker build --build-arg SDK_MANAGER_VERSION=1.0.0-5517 -t jetpack .
```

Comment (graugans) [reply README.md:15]: I've updated the PR

**Review by atinfinity — APPROVED** [review]

**Pull request merged**